mk/dev.mk

```diff
@@ -63,6 +63,7 @@ GOLANGCI_LINT=$(CI_TOOLS_BIN_DIR)/golangci-lint
 HELM_DOCS=$(CI_TOOLS_BIN_DIR)/helm-docs
 KUBE_LINTER=$(CI_TOOLS_BIN_DIR)/kube-linter
 HADOLINT=$(CI_TOOLS_BIN_DIR)/hadolint
+YAMLLINT=yamllint
```

Comment:
We don't want to use global binaries, we have to figure out the way to put it inside `$(CI_TOOLS_BIN_DIR)`